fpdf/enums.py

```diff
@@ -574,5 +574,12 @@ class PDFStyleKeys(Enum):
     STROKE_DASH_PATTERN = Name("D")  # array of array, number, e.g. [[1 1] 0]
 
 
+class Corner(CoerciveEnum):
+    TOP_RIGHT = Name("TOP_RIGHT")
```

Comment:
`Name()` should only be used for string values that are directly embedded in the PDF.
Here you can just use a string ( `"TOP_RIGHT"` )

Reply:
Fixed